src/workerd/api/tests/streams-test.js

```diff
@@ -317,31 +301,25 @@ export const tsCancel = {
     // readable is canceled or the writable is aborted. Verify also that
     // errors thrown by the cancel function are propagated.
     {
-      let cancelCalled = false;
-      const { readable } = new TransformStream({
-        async cancel(reason) {
-          strictEqual(reason, 'boom');
-          await scheduler.wait(10);
-          cancelCalled = true;
-        },
```

Comment:
Do these negative diffs retain coverage I don't see the `cancelCalled` variable retained in the wd-test?

Reply:
While this is a net improvement to the test, I would have liked to see these types of changes deferred to either a separate commit or a separate PR as they make reconciling the ported tests to the original tests more difficult. Specifically, as @guybedford points out, it makes it more difficult to know that the coverage has remained the same when comparing the original and new side-by-side. Consider it non-blocking tho.

Reply:
That doesn't really make sense. the todo doc could have been removed from the original commit without flattening all of the changes to the non-related files. Also, these changes could have been deferred until the main set was landed. Either way, it's done now. Let's make sure it's a bit clearer moving forward as we do need to be able to reconcile with the original tests.

Reply:
FYI I reverted this change.